docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -223,6 +223,25 @@ max-trailing-peers=2
 Specifies the maximum P2P peer connections for peers that are trailing behind the local chain head. 
 The default is unlimited but the number of trailing peers cannot exceed the value specified by [`--max-peers`](#max-peers).
 
+### metrics-enabled
+
+```bash tab="Syntax"
+--metrics-enabled
+```
+
+Set to `true` to enable the [Prometheus](https://prometheus.io/) monitoring service to access Pantheon client metrics.
```

Comment:
Suggest changing 'Pantheon client' to 'Pantheon' to be consistent with rest of documentation

Reply:
I think we need a link to the conceptual debugging content as well - otherwise if they come here first, they won't know it exists.

Reply:
Done